### PR TITLE
Altered Naming Convention for Duplicates

### PR DESCRIPTION
**Pull request by Ammelll**
In my experience the current naming convention of "Copy of {AutoName}" is very inconvenient, primarily when sorting the Path/Auto names alphabetically as duplicating an auto will send it to a random place in the stack instead of right next to the Path/Auto being duplicated. I believe the changes I've proposed would address this.

### CODE REVIEW

**Comment by rjbell4** [comment]
I concur that something that appends a suffix would be preferred over something the prepends a prefix, as it would result in less scrolling when creating new paths or autos by duplicating something that exists and then modifying it.

**Comment by mjansen4857** [comment]
Just checking in on the status of this as it still has failing tests and needs to be formatted. 

You can format by running `dart format .` in the root of the project directory.

**Comment by mjansen4857** [comment]
It looks like the tests are failing because the actual naming logic is not working as expected. For example, when duplicating `Example Path (1)` it expects to see `Example Path (2)` but doesn't. I'm assuming its creating `Example Path (1) (1)`

File: lib/pages/project/project_page.dart [hunk 1]
```diff
@@ -908,9 +908,17 @@ class _ProjectPageState extends State<ProjectPage> {
         for (PathPlannerPath path in _paths) {
           pathNames.add(path.name);
         }
-        String pathName = 'Copy of ${_paths[i].name}';
+        String pathName = _paths[i].name;
+        RegExp exp = RegExp(r'\(\d+\)');
+        String source = pathName.substring(pathName.length - 3);
         while (pathNames.contains(pathName)) {
-          pathName = 'Copy of $pathName';
+          if (exp.hasMatch(source)) {
```
Comment (mjansen4857) [inline lib/pages/project/project_page.dart:915]: I'm assuming tests are failing because you're matching the regex against the substring with the `(x)` removed. I think this should be changed to match against the entire path name and then only do the substring if it has a match. Same thing for the autos.

Comment (mjansen4857) [reply lib/pages/project/project_page.dart:915]: When fixing this, could you also add on to the duplication tests to duplicate them a third time. i.e. check for `Example Path (3)` just to be extra sure the increment works.

**Comment by Ammelll** [comment]
@mjansen4857 anything holding this up?

**Comment by Ammelll** [comment]
@mjansen4857 bump
